### DISCUSSION TITLE
Where can I see an example of CQL-Text filter in the GUI?

### DISCUSSION THREAD

**Discussion by chiarch84** [radiantearth/stac-browser#378] [Q&A] [answered]
I see that the CQL-Text filter extension should be implemented in the browser. Where can I see an example of it? Do i just need to set the conformance class in order to see an enriched filter GUI?
Thanks

**Comment by m-mohr** [comment, marked as answer]
You need to follow the corresponding extensions.
If they are correctly implemented, you should see a free-text search field in Collection, Global Item or Collection-based Item Search.
The extensions are:
- https://github.com/cedadev/stac-freetext-search
- https://github.com/stac-api-extensions/collection-search/

You can find a live example here: https://radiantearth.github.io/stac-browser/#/search/external/emc.spacebel.be/

**Comment by chiarch84** [comment]
Thanks @m-mohr, so the filter extension is only implemented for collection search and not for items search, is this correct?

**Reply by m-mohr** [reply]
No:

> you should see a free-text search field in Collection, Global Item or Collection-based Item Search

**Reply by chiarch84** [reply]
hmm, so maybe I'm not getting something. We have implemented the filter extension and put it in the conformance classes, but I don't see any changes in the filtering GUI. Is it mandatory to parallelly implement also the collection search?

**Reply by m-mohr** [reply]
No, it should work independantly. If it doesn't show up there's probably a conformance class missing or wrong.
It works for Spacebel, so maybe compare to their implementation.
